docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    command: >
+      sh -c "python manage.py wait_for_db &&
+             python manage.py migrate && 
+             python manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - ./:/app

```

Comment:
Do not mount the whole directory as a volume in the final version in the future, it should be done during development only